crates/invoker-impl/src/metric_definitions.rs

```diff
@@ -26,26 +27,8 @@ pub(crate) fn describe_metrics() {
     );
 
     describe_counter!(
-        INVOKER_INVOCATION_TASK_STARTED,
-        Unit::Count,
-        "Number of active invocation tasks started"
-    );
-
-    describe_counter!(
-        INVOKER_INVOCATION_TASK_FAILED,
-        Unit::Count,
-        "Number of active invocation tasks failed"
-    );
-
-    describe_counter!(
-        INVOKER_INVOCATION_TASK_SUSPENDED,
-        Unit::Count,
-        "Number of active invocation tasks suspended"
-    );
-
-    describe_gauge!(
-        INVOKER_INFLIGHT_INVOCATIONS,
```

Comment:
W/o this gauge we no longer know how many invocations are currently executed by the invoker, right? Isn't this a useful metric to have or why did you remove this metric?

Reply:
Yes, it's a useful metric but unfortunately it's broken (shows mostly negative results in my tests) with the current implementation. We should re-introduce it again but it needs to cover the correct spots to increment and decrement.

Reply:
Thx for the clarification.